packages/types/src/workflow/cart/update-cart.ts

```diff
@@ -0,0 +1,10 @@
+export interface UpdateCartWorkflowInputDTO {
```

Comment:
nit: I believe I introduced cart workflow types in `types/src/cart/workflows.ts`. Perhaps, we can consolidate. Nothing important, so consider this non-blocking